lectures/aiyagari_jax.md

```diff
@@ -29,16 +29,16 @@ The model features
 
 The Aiyagari model has been used to investigate many topics, including
 
-- precautionary savings and the effect of liquidity constraints [[Aiy94](https://python.quantecon.org/zreferences.html#id137)]
-- risk sharing and asset pricing [[HL96](https://python.quantecon.org/zreferences.html#id129)]
-- the shape of the wealth distribution [[BBZ15](https://python.quantecon.org/zreferences.html#id130)]
+- precautionary savings and the effect of liquidity constraints [[Aiy94](https://python.quantecon.org/zreferences.html#id138)]
```

Comment:
@HengchengZhang I am going to merge this to fix the links in the short term but we need to migrate all these links to use proper `cite` roles such as: https://manual.quantecon.org/styleguide/references.html